testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -166,7 +170,7 @@ public void testNotUsername() {
             RealmModel realmModel = session.getContext().getRealm();
             PasswordPolicyManagerProvider policyManager = session.getProvider(PasswordPolicyManagerProvider.class);
 
-            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, "notUsername"));
+            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, NotUsernamePasswordPolicyProviderFactory.ID));
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done